builds/azure-pipelines/template-steps-build-test.yml

```diff
@@ -9,6 +9,23 @@ steps:
   inputs:
     version: '3.1.x'
 
+- script: npm install -g azure-functions-core-tools@3 --unsafe-perm true
```

Comment:
Why the unsafe perm param?

Reply:
Removed (copied from the Functions CLI docs, not sure why it's set there)